Structure_Engine/Create/Elements/PadFoundation.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.ComponentModel;
+using BH.oM.Base.Attributes;
+using BH.oM.Geometry;
+using BH.oM.Geometry.CoordinateSystem;
+using BH.oM.Spatial.ShapeProfiles;
+using BH.oM.Structure.Elements;
+using BH.oM.Structure.SurfaceProperties;
+using BH.Engine.Base;
+using BH.Engine.Geometry;
+using BH.Engine.Spatial;
+
+
+namespace BH.Engine.Structure
+{
+    public static partial class Create
```

Comment:
Curious if it would not be worth having `Create` methods taking following inputs:
- outlines as a collection of `ICurve`
- thickness property as `double` and `IMaterialFragment`

I will never use these so please fill free to ignore, just trying to put myself in the shoes of a GH user

Reply:
I think it's quite similar to the first `Create` method, can always be added in the future.